src/UI/Implementation/Component/Image/Image.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Image;
+
+use ILIAS\UI\Component as C;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
+
+/**
+ * Class Image
+ * @package ILIAS\UI\Implementation\Component\Image
+ */
+class Image implements C\Image\Image {
+	use ComponentHelper;
+
+	/**
+	 * @var	string
+	 */
+	private $type;
+
+	/**
+	 * @var	string
+	 */
+	private  $src;
+
+	/**
+	 * @var	string
+	 */
+	private  $alt;
+
+	/**
+	 * @var []
+	 */
+	private static $types = [
```

Comment:
Yeah! Just noticed we can use those now, as we dropped PHP 5.3 support for ILIAS 5.2. Nice =)